django_x509/base/models.py

```diff
@@ -203,14 +203,14 @@ def save(self, *args, **kwargs):
         generate = False
         if not self.pk and not self.certificate and not self.private_key:
             generate = True
-        super().save(*args, **kwargs)
+            
         if generate:
```

Comment:
```suggestion
```
Looks like we can simplify this further: remove these lines and any reference to the `generate` variable.